Updating to sdk's for release

mhoeger opened the pull request:
Updating nugets generated from azure-webjobs-sdk to be pulled into script for a prod release

brettsam commented:
I checked in a CosmosDBTrigger fix today and *just* published it to MyGet. I saw the alert for this PR right after I hit "Deploy" -- so it probably went up after you looked.

mhoeger commented:
Ahh, ok! Will do!

brettsam commented on src/WebJobs.Script.Host/packages.config:
```diff
@@ -19,15 +19,15 @@
+  <package id="Microsoft.Azure.WebJobs.Extensions.NotificationHubs" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.SendGrid" version="2.2.0-beta1-10588" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.Twilio" version="2.2.0-beta1-10588" targetFramework="net471" />
   <package id="Microsoft.Azure.WebJobs.Logging.ApplicationInsights" version="2.2.0-beta1-11229" targetFramework="net471" />
```
A few of these are incorrect. Make sure that each of the previous major versions still matches after the update. E.G. DocumentDB should be 1.2.0-beta1-10588.

fabiocav commented on src/WebJobs.Script.Host/WebJobs.Script.Host.csproj:
```diff
@@ -105,43 +105,43 @@
     </Reference>
     <Reference Include="Microsoft.Azure.WebJobs.Extensions.MobileApps, Version=1.1.0.0, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.MobileApps.1.1.0-beta4-10582\lib\net45\Microsoft.Azure.WebJobs.Extensions.MobileApps.dll</HintPath>
+      <HintPath>..\..\packages\Microsoft.Azure.WebJobs.Extensions.MobileApps.2.2.0-beta1-10588\lib\net45\Microsoft.Azure.WebJobs.Extensions.MobileApps.dll</HintPath>
```
These are major version updates. We want to stick with the latest from the same major version release.

fabiocav commented on src/WebJobs.Script.Host/packages.config:
```diff
@@ -17,19 +17,19 @@
+  <package id="Microsoft.Azure.WebJobs.Extensions.ApiHub" version="2.2.0-beta1-10588" targetFramework="net471" />
   <package id="Microsoft.Azure.WebJobs.Extensions.BotFramework" version="1.0.15-beta" targetFramework="net471" />
-  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="1.1.0-beta4-10582" targetFramework="net471" />
+  <package id="Microsoft.Azure.WebJobs.Extensions.DocumentDB" version="2.2.0-beta1-10588" targetFramework="net471" />
```
Review major version updates.

fabiocav commented:
One more... can you make sure Event Grid is updated to 1.0.0-beta3-10020

mhoeger commented:
Thanks for catching the errors! I did a blanket find/replace and didn't notice that sdk-extensions nuget packages had a different major/minor version. Updated and added event grid update too
